### DISCUSSION TITLE
[❄️ Snowflake Official] A Guide on creating GitHub issues + FAQ + Commonly Known Issues

### DISCUSSION THREAD

**Discussion by sfc-gh-jcieslak** [Snowflake-Labs/terraform-provider-snowflake#2790] [Announcements]
Hey 👋
Recently, we added a new document to our repository ([here](https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/CREATING_ISSUES.md)). This document aims to guide you on how to create a new GitHub issue in our repository, ensuring a smoother and more efficient issue reporting process.

In addition to providing step-by-step instructions on issue creation, the document also includes a FAQ section. This section has been designed to help you quickly find answers to frequently asked questions, saving you time and effort.

Furthermore, we have included a section on commonly known issues. This section highlights some of the most common bugs and their resolutions, allowing you to troubleshoot and resolve issues independently.

We highly encourage everyone to take a moment to read through this new document before creating a GitHub issue. By familiarizing yourself with the process and referring to the FAQ and Commonly Known Issues sections, you may find that your questions or concerns have already been addressed.

**Comment by chrisweis** [comment]
This is great! Thanks so much! 😁
